sklearn/metrics/tests/test_common.py

```diff
@@ -834,6 +863,8 @@ def test_raise_value_error_multilabel_sequences(name):
 
 @pytest.mark.parametrize('name', METRICS_WITH_NORMALIZE_OPTION)
 def test_normalize_option_binary_classification(name):
+    if name in METRIC_UNDEFINED_BINARY:
```

Comment:
Why do we need it? Same comment below.

Reply:
We don't, but it's sensible for the future??

Reply:
Hmm, I guess it's sensible to refactor the common test for metrics in the future. Some part of it seems not so good IMO :)